src/Z3/Float.extension.st

```diff
@@ -0,0 +1,15 @@
+Extension { #name : #Float }
+
+{ #category : #'*Z3' }
+Float >> toFloatingPoint: length [
+
+	| bits ctxt sort |
+	length = 64 ifFalse: [ self error: 'bit length mismatch' ].
+
+	bits := self asIEEE64BitWord toBitVector: length.
```

Comment:
`asIEEE64BitWord` here is causing failures on P8 and on St/X. On Pharo one can simply inline `asIEEE64BitWord`
into this method like 

```
bits := ((self basicAt: 1) bitShift: 32) + (self basicAt: 2)
bits := bits toBitVector: length.
```

but that does not solve the problem for St/X...I have to think.